Update 'Customize built-in policies' tutorial (#2599)
elastic/docs-content: manage-data/lifecycle/index-lifecycle-management/tutorial-customize-built-in-policies.md

* `traces@lifecycle`
* `traces-otel@lifecycle`

> This tutorial gives the example of customizing how ingested logging data is managed. Rather than use the default lifecycle settings from the built-in `logs@lifecycle` {{ilm-init}} policy, you can use the **Index Lifecycle Policies** feature in {{kib}} to tailor a new policy based on your application’s specific performance, resilience, and retention requirements. This involves three main steps:

Reviewer: Minor suggestion to simplify:

```suggestion
This tutorial covers customizing the way ingested logging data is managed. Rather than use the default lifecycle settings from the built-in `logs@lifecycle` {{ilm-init}} policy, you can use the **Index Lifecycle Policies** feature in {{kib}} to tailor a new policy based on your application’s specific performance, resilience, and retention requirements. This involves three main steps:
```

Author: nice catch!
